modules/enabling-automatic-stream-sharding.adoc

```diff
@@ -0,0 +1,50 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-09-22
+:_mod-docs-content-type: PROCEDURE
+
+[id="enabling-automatic-stream-sharding_{context}"]
+= Enabling automatic stream sharding
+
+Loki provides automatic stream sharding mechanism. Automatic stream sharding helps with making Loki more resilient without having to configure additional stream labels. To enable automatic stream sharding, remove `perStream*` attributes under the limits part of the spec. 
+
+.Prerequisites
+* You have configured {loki-op}.
+
+.Procedure
+* Remove definitions of the `perStream*` attributes.
++
+.Example CR with perStream* attributes
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
```

Comment:
It's better we mention just one LokiStack spec for stream sharding both globally and tenant wise

```
spec:
  limits:
    global:
      ingestion:
        perStreamDesiredRate: 5
    tenants:
      application:
        ingestion:
          perStreamDesiredRate: 6
  tenants:
    mode: openshift-logging
....
```

Reply:
done

Reply:
Explained below. We should only mention perStreamDesiredRate for tenants since Loki enables automatic stream sharding by default (3MB)